ARAnalytics.m

```diff
@@ -537,6 +537,18 @@ + (void)pageView:(NSString *)pageTitle
     }];
 }
 
++ (void)pageView:(NSString *)pageTitle withProperties:(NSDictionary *)properties
+{
+    if (!pageTitle) return;
```

Comment:
Happy with an assert ( double check if thats a pattern we use in ARAnalytics please)